include/unk_02097720.h

```diff
@@ -0,0 +1,15 @@
+#ifndef HEARTGOLD_UNK_02097720_H
+#define HEARTGOLD_UNK_02097720_H
+
+#include "task.h"
+
+typedef struct Unknown_02097720 {
+    u32 unknown_0;
+    u16* unknown_1;
```

Comment:
Field names, if unknown, should reflect the byte offset within the struct (in this case, 4).